source/driver-vs-mongoose.txt

```diff
@@ -0,0 +1,103 @@
+them as it is similar to working with a relational database. 
+
+Schema Validation
+-----------------
+
+Both the {+driver-long+} and Mongoose let you apply schema validation
+to your collection. However, the biggest difference between the two are
+the way they each set and manage schema validation.
+
+The {+driver-long+} applies the schema validation at the database
+level. In comparison, Mongoose applies schema validation at the application level. In the
+{+driver-short+} schema validation works on individual collections by
+using either `JSON Schema Validation <https://www.mongodb.com/docs/manual/core/schema-validation/specify-json-schema/>`__
+or `Schema Validation with Query Operators <https://www.mongodb.com/docs/manual/core/schema-validation/specify-query-expression-rules/>`__.
+Validation with query operators is useful for dynamic validation rules
+calculated at runtime. By contrast, JSON validation is useful for, among other
+reasons, being an open standard which makes it easy to connect to
+third-party libraries.
+
+Mongoose applies the schema validation in a twofold procedure; firstly by telling your
+application what data types and fields are allowed to be in a particular
+collection, then it confirms the data matches the set criteria. Mongoose's
+schema validation works on the application level, meaning MongoDB
+doesn't know it exists.
+
+.. tip::
+   
+   Though Mongoose can be more efficient to create schemas,
+   the {+driver-long+} is generally twice as fast at reading and writing
+   operations. This means for larger, more performance-intensive
+   projects the {+driver-short+} is likely the better option.
+
+Similarities
+------------
+
+As Mongoose is built upon the {+driver-long+}, they share many
+capabilities. Even if you use Mongoose, you always have the option to
+use any native function or command built into the {+driver-short+}
+alongside Mongoose. Mongoose also offers its own methods that are quite
+similar to the {+driver-short+}, such as ``updateOne()`` and
+``replaceOne()``, though these methods work differently under the hood.
+
+Differences
+-----------
+Though MongoDB was designed with a flexible data model in mind, Mongoose
+forces a semi-rigid ``schema`` from the beginning. With Mongoose, developers
+must define a ``schema`` and ``model``. Moreover, the handling of read
+and write operations differs between the two, resulting in the
+{+driver-short+} generally running twice as fast as Mongoose. 
+
+.. tip::
+   
+   There are many other Mongoose specific methods not discussed. For
+   more information on how to use Mongoose, check out the official `Mongoose
+   documentation <https://mongoosejs.com/docs/>`__.
+
+Conclusion
+----------
+
+Both Mongoose and the {+driver-long+} offer comparable features
+and capabilities. While Mongoose helps ease MongoDB development for
+newcomers, it employs various automated processes behind the scenes that
+might lead to unforeseen consequences down the line.
```

Comment:
>  While Mongoose helps ease MongoDB development for
newcomers, it employs various automated processes behind the scenes that
might lead to unforeseen consequences down the line.

This is framing mongoose as a risk that developers should avoid. Mongoose is part of the [Verified Solutions Program](https://www.mongodb.com/blog/post/mongodb-announces-new-verified-solutions-program) which implies customers with commercial support can officially use mongoose and receive support for their applications.

The conclusion should be used to promote the value of mongoose as a product while calling out that it adds a layer of abstraction above the driver - not that it will result in a negative experience.

Reply:
It might be worth adding a link to the [Mongoose Learning Byte](https://learn.mongodb.com/learn/course/mongoose-odm-with-mongodb/learning-byte/learn) as well